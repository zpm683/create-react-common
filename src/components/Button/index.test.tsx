import { Button } from ".";
import { render } from "@testing-library/react";

describe("Button unit test", () => {
  it("Snapshot", () => {
    const { container } = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
