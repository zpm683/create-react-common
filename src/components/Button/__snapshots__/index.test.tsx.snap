// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button unit test Snapshot 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</div>
`;
